coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -97,6 +99,22 @@ def value_inference(self):
 
 @register_op(doc_str="")
 class const(Operation):
+    """
+    Return constant values represented as ``ImmediateValue`` or ``FileValue``
+    in the NNv2 proto.
+    
+    Parameters
+    ----------
+    val: const<*,T> (Required)
```

Comment:
Another parameter is `mode` which is optional and defaults to "immediate_value". It can take two values, "immediate_value" or "file_value" which determines how the constant value is stored in the internal MIL format. Generally for a large constants (such as convolution weights), "file_value" is used, whereas for smaller size constants (such as values of a stride etc), "immediate_value" is used.

Reply:
Please review this snippet:

> mode: immediate_value, file_value (Optional)
>     	* Determines how the constant value is stored in the internal MIL format. For  large constants such as convolution weights, use ``file_value``. For smaller-size constants such as values of a stride, use ``immediate_value``.